libs/books/feature/src/lib/book-search/book-search.component.html

```diff
@@ -48,7 +49,9 @@
 <ng-template #empty>
   <div class="empty">
     <p>
```

Comment:
It would be good fetch error (API failure) and show message as "No results found"